docs/pages/concepts/logical/relationships.md

```diff
@@ -1,11 +1,12 @@
 ---
 layout: default
 title: Relationships
-permalink: concepts/relationshps
+permalink: concepts/relationships
```

Comment:
Good catch, @captain-Akshay. Please run a full search across all docs to ensure that any links to this page are also updated. Especially, the toc.yaml.